tests/foreman/virtwho/utils.py

```diff
@@ -139,11 +139,15 @@ def get_virtwho_status():
     """Return the status of virt-who service, it will help us to know
     the virt-who configure file is deployed or not.
     """
+    _, logs = runcmd('cat /var/log/rhsm/rhsm.log')
+    error = len(re.findall(r'\[.*ERROR.*\]', logs))
     ret, stdout = runcmd('systemctl status virt-who')
     running_stauts = ['is running', 'Active: active (running)']
     stopped_status = ['is stopped', 'Active: inactive (dead)']
     if ret != 0:
         return 'undefined'
```

Comment:
this is not necessary.  what's your opinion?

Reply:
I think we should still return sth. when ret!=0, which means run command failed, maybe replace it to 'runcmderror' or other.